feature-detects/img/loading.js

```diff
@@ -0,0 +1,19 @@
+/*!
+{
+  "name": "image native lazy loading",
+  "property": "loading",
+  "caniuse": "loading",
```

Comment:
the caniuse property "loading-lazy-attr" (https://caniuse.com/#feat=loading-lazy-attr)